course_discovery/apps/edly/api/v1/serializers.py

```diff
@@ -0,0 +1,96 @@
+"""
+Serializers for Edly API.
+"""
+from course_discovery.apps.api.fields import StdImageSerializerField
+from rest_framework.serializers import (
+    ChoiceField,
+    IntegerField
+)
+from course_discovery.apps.course_metadata.models import (
+    Course,
+    Organization,
+    Program
+)
+from course_discovery.apps.core.models import Partner
+from course_discovery.apps.api.serializers import (
+    CourseSerializer,
+    CourseRunSerializer,
+    CorporateEndorsementSerializer,
+    DegreeSerializer,
+    EndorsementSerializer,
+    FAQSerializer,
+    MinimalPersonSerializer,
+    OrganizationSerializer,
+    ProgramSerializer,
+    SubjectSerializer,
+    VideoSerializer
+)
+
+
+class EdlyProgramSerializer(ProgramSerializer):
+    status = ChoiceField(choices=['active', 'unpublished'])
+    individual_endorsements = EndorsementSerializer(many=True, allow_null=True, required=False)
+    subjects = SubjectSerializer(many=True, allow_null=True, required=False)
+    video = VideoSerializer(allow_null=True, required=False)
```

Comment:
just a question, what is this video about? a more verbose name might be used to depict the purpose of this video.

Reply:
Trying to stick with OpenEdx names. Video is a video that can be attached to programs. Although we won't use it just making it optional here.